appveyor.yml

```diff
@@ -21,7 +21,13 @@ install:
   - del rustup-init.exe
   - rustc -vV
   - cargo -vV
+  - rustup component add llvm-tools-preview --toolchain stable-x86_64-pc-windows-msvc
+  - cargo install cargo-llvm-cov
 
 test_script:
   - cargo test
+  - cargo llvm-cov --html
```

Comment:
does this run "test" again? test + cov. if so we should run it only with cov, build time is limited on appveyor.

Reply:
According to the `llvm-cov` [README](https://github.com/taiki-e/cargo-llvm-cov#:~:text=By%20default%2C%20run%20tests%20(via%20cargo%20test)%2C%20and%20print%20the%20coverage%20summary%20to%20stdout.) it does run `cargo test`. Fixed to just run `cargo llvm-cov` to save build time.